terraso_backend/apps/soil_id/models/soil_data.py

```diff
@@ -205,8 +204,14 @@ class Grazing(models.TextChoices):
         WILDLIFE_GRASSLANDS = "WILDLIFE_GRASSLANDS", "Wildlife (grasslands, giraffes, ibex)"
 
     grazing_select = models.CharField(blank=True, null=True, choices=Grazing.choices)
+
+    class SoilDataDepthIntervalPreset(models.TextChoices):
+        LANDPKS = "LANDPKS"
+        NRCS = "NRCS"
+        CUSTOM = "CUSTOM"
```

Comment:
question: should these constants be  stored elsewhere?

Reply:
I think it's fine to put them here, following the pattern for other enums